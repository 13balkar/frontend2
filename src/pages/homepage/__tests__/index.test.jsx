import * as React from 'react';
import { render } from '@testing-library/react';
import Homepage from '../';
jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Homepage', () => {
  it('should render the homepage', () => {
    const { asFragment } = render(<Homepage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
