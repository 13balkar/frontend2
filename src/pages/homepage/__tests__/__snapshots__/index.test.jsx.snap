// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should render the homepage 1`] = `
<DocumentFragment>
  <div
    class="homepage"
  >
    <div
      class="login-image"
    >
      <div
        class="content"
      >
        <h1>
          Design APIs fast,
        </h1>
        <h1>
          Manage content easily
        </h1>
      </div>
      <img
        alt=""
        src="assets/background.png"
      />
    </div>
    <div
      class="login-box"
    >
      <h1>
        Login to your CMS+ account
      </h1>
      <div
        class="card"
      >
        <label>
          Email
        </label>
        <input
          type="text"
          value=""
        />
        <label>
          Password
        </label>
        <input
          type="password"
          value=""
        />
        <button>
          Login
        </button>
        <div
          class="links"
        >
          <p>
            Forgot password?
          </p>
          <p>
            Create an account
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
